docs/glossary.md

```diff
@@ -0,0 +1,265 @@
+
+#### Consensus rules
+
+Rules, encoded in software, that allow a network of nodes to reach an agreement about the state of the ledger. Rules include what data blocks can contain, how block data is structured, and how nodes validate blocks. 
+
+#### Consensus rules voting
+
+Changes to Decred's consensus rules can only be made through an [on-chain voting process](https://docs.decred.org/getting-started/user-guides/agenda-voting/) which lasts for around one month. The change is made only if at least 75% of votes are in favor. 
+
+#### Constitution
+
+A [document](https://docs.decred.org/getting-started/constitution/) which defines the purpose and guiding principles of the Decred project.
+
+#### Credits
+
+The full unit of the Decred currency (i.e. 1 DCR).
+
+#### DCR
+
+Ticker for the Decred currency.
+
+#### Decred Change Proposal (DCP)
+
+A [design document](https://github.com/decred/dcps) that describes potential protocol or consensus changes to Decred. DCPs primarily serve for documentation, fostering cross-implementation compatibility and helping ensure proper engineering rigor is followed.
+
+#### Decrediton
+
+A GUI (Graphical User Interface) [Decred wallet](https://github.com/decred/decrediton/) maintained by the core team. 
+
+#### Difficulty
+
+Difficulty is a measure of how difficult it is to mine a new block (i.e. find a hash below a given target). In Decred, the PoW difficulty is calculated from the exponentially weighted average of differences in previous block times.  
+
+#### Distributed ledger
+
+A cryptographically secure ledger, composed of nodes in a network, where each node holds a copy of the same ledger. 
+
+#### Expired ticket
+
+Tickets that reached the end of their window without being called to vote - these can be revoked, but do not grant a reward.
+
+#### Faucet
+
+A mechanism for obtaining free (testnet) tokens.
+
+#### Hash
+
+The output of a cryptographic hashing function that produces a fixed-size and unique hash value from variable-size input. 
+
+#### Hash function
+
+A cryptographic function that produces a fixed-size and unique hash value from variable-size transaction input. Decred uses the [BLAKE-256](https://docs.decred.org/research/blake-256-hash-function/) hashing function. 
+
+#### Hashrate
+
+The number of hashes per second computed by miners on the network.
+
+#### Hybrid PoW/PoS
+
+In Decred, a [hybrid PoW/PoS system](https://docs.decred.org/research/hybrid-design/) is used, whereby blocks mined by PoW miners must be approved by ticket holders, which are pseudorandomly selected from the ticket pool. This provides a check on PoW miners and increases the overall cost of attacking the network. When a block is approved, 60% of the block reward goes to the PoW miner, 30% goes to the holders of  tickets called to vote (approve blocks), and 10% goes to the Decred Treasury to fund the project's development.
```

Comment:
Instead of saying "the project's development" I would just suggest "the project".  While I realize that all facets can be considered a form of "development", most people interpret that to mean writing code, and the Treasury is used for more than writing code.

Reply:
Agreed. Reworded with suggestion.

#### Hybrid PoW/PoS

In Decred, a [hybrid PoW/PoS system](https://docs.decred.org/research/hybrid-design/) is used, whereby blocks mined by PoW miners must be approved by ticket holders, which are pseudorandomly selected from the ticket pool. This provides a check on PoW miners and increases the overall cost of attacking the network. When a block is approved, 60% of the block reward goes to the PoW miner, 30% goes to the holders of  tickets called to vote (approve blocks), and 10% goes to the Decred Treasury to fund the project.